src/radix.py

```diff
@@ -9,8 +9,22 @@ def count_sort(x: str) -> str:
     >>> count_sort('')
     ''
     """
-    return ""
 
+    count = {}
+    for i in range(len(x)):
+        if x[i] in count:
+            count[x[i]].append(i)
+        else:
+            count[x[i]] = [i] # {'a':[2,3]}
+    
+    output = ''
+    count = OrderedDict(sorted(count.items()))
```

Comment:
When you use `sorted` here, you are using $O(\sigma\log\sigma)$ time. When we assume $\sigma\in O(1)$ that is of course not a problem, but when the alphabet is dynamic, you need to be more careful. If we just get strings, this is the best that we can do, but in something like Skew or SAIS, this would break the running time.